portfolio/src/main/webapp/script.js

```diff
@@ -22,7 +22,21 @@ $(document).ready(function() {
 
     let $height = $("header").outerHeight(true);
 
-    $("#main-container").css("margin-top", `${$height + 20 + "px"}`);
+    $("#main-container").css("margin-top", `${$height + 30 + "px"}`);
+
+    let threshold = $(document).height() * 0.1;
+    let didShowDialog = false;
+
+    $(window).scroll(function() {
+        if($(window).scrollTop() + $(window).height() > $(document).height() - threshold && !didShowDialog) {
+            $("#dialog").show();
+            didShowDialog = true;
+        }
+    });
```

Comment:
It took me a little bit of head scratching to understand what's going on here.  I think what you're trying to do is notice when they're about 90% of the way through your page and then you'll show them a dialog telling them to check out your other site, right?

Perhaps break this functionality into a helper method, rather than putting it up next to setting the copyright date?  Or perhaps just add a comment over the block of code explaining the general idea?  I'm open to lots of possibilities here!

Reply:
Thanks I'll add the comments